labgrid/environment.py

```diff
@@ -22,7 +22,7 @@ def __attrs_post_init__(self):
             self.config = Config(self.config_file)
         except:
             raise NoConfigFoundError(
-                "{} is not a valid yaml file".format(self.config_file)
```

Comment:
The Config class already returns NoConfigFoundError or InvalidConfigError, so we shouldn't catch those here only to give less information.

Reply:
Ok, that was my second feeling on these lines thus I will remove the re-raise.